its/ruling/src/test/resources/regex-examples/java-S5852.json

```diff
@@ -1,6 +1,7 @@
 {
 'org.regex-examples:regex-examples:src/main/java/org/regex/examples/RegexDatabase1.java':[
 33,
+49,
```

Comment:
The differences in this rule are due to the fix made on Analyzer Commons side: https://github.com/SonarSource/sonar-analyzer-commons/pull/159.

I double-checked, and the change makes sense: we are now correctly able to determine when a regex is anchored at end.  The new issues looks like new true positives!